templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -1073,7 +1074,7 @@ Resources:
       Description: >-
         Allow SSH from the Linux bastion host / bootstrap instance to the EKS
         nodes.
-      GroupId: !Ref LinuxBastionSecurityGroup
+      GroupId: !Ref QSBastionSecurityGroup
```

Comment:
*nit*/*non-blocking*:
I guess we give the windows bastion host pretty much the same "permissions" now. Technically, I guess the windows bastion host would only need access to the k8s API and to the LB, and ie. not SSH. You could argue, the same is true for the linux bastion host.
Having said that, I can see that unifying the bastion hosts sec groups being easier to maintain.

Reply:
```
- Created common QSBastionIamRole & QSBastionSecurityGroup to be used for both Ubuntu and Windows Bastion.
- Separated LinuxBastionSecurityGroup (for ssh) & WindowsBastionSecurityGroup (for RDP) have specific ports used for Ubuntu and Windows Bastion.
```

As said above,

- EC2 instance needs an IAM role to access all clusters (single and multi). Hence created a common QSBastionIamRole for both EC2 instances. 
- The ingress/egress SG rules need to be the same for both EC2 to access the K8s control plane /API.  Hence create a common QSBastionSecurityGroup.
- Separated LinuxBastionSecurityGroup (for ssh) & WindowsBastionSecurityGroup (for RDP) have specific ports used for Ubuntu and Windows Bastion.  We are not adding SSH to Windows or RDP to Linux. Hope that helps to clarify.